TASVideos.Parsers/Parsers/Ctm.cs

```diff
@@ -0,0 +1,38 @@
+using TASVideos.MovieParsers.Result;
+
+namespace TASVideos.MovieParsers.Parsers;
+
+[FileExtension("ctm")]
+internal class Ctm : ParserBase, IParser
+{
+	private const decimal FrameRate = 59.83122493939037M;

```

Comment:
Perhaps instead do `268111856.0M / 4481136.0M` and link to https://github.com/citra-emu/citra/blob/a2f34ea82b5a31a7e842d0099921b85b8bce403f/src/core/hw/gpu.h#L27 in a comment.

Reply:
And don't use a url shortener please...